examples/guided.py

```diff
@@ -3,6 +3,10 @@
 from archinstall.lib.hardware import hasUEFI
 from archinstall.lib.profiles import Profile
 
+if hasUEFI() is False:
```

Comment:
This commit should NOT be in 2.2.0. Which is why rebasing these commits is so important. This breaks MBR/GRUB in the 2.2.0 branch

Reply:
I should pull in master and fix this, this is bound to happen more times. Give me an hour.

Reply:
yeah this breaks my implementation of grub/bios support. would you mine rebasing to  the 2.2.0 branch to fix this?